### PR TITLE
fix(get-started): move wallet setup + add network setup

### PR DESCRIPTION
**Pull request by justprosh**
I am not sure about `Consider reading: [Keys management with Fluence CLI](cli-keys-management.md)`, because it is totally out of context everywhere in this tutorial

### CODE REVIEW

File: docs/build/get-started.mdx [hunk 1]
```diff
@@ -671,6 +493,188 @@ After `fluence build`, re-run `cargo test --workspace` and voila, all is well ag
 
 Now, we're ready to talk about Deal deployment and Compute Marketplace.
```
Comment (folex) [inline docs/build/get-started.mdx]: Aaand it doesn't talk about that any more. 
```suggestion
Now, before we're ready to move on to Deployment to Marketplace, let's go through Wallet setup and get tokens necessary for the Deployments.
```

And move `Now, we're ready to talk about Deal deployment and Compute Marketplace.` to a later point.

File: docs/build/get-started.mdx [hunk 2]
```diff
@@ -270,6 +81,17 @@ Choose the *minimal* scaffolding option, and press return. When prompted, enter
...
 ```
 
+Also, we can choose across different networks to deploy our services. For now, we will use the default network, which is `kras`:
+
+```bash
+? Select Fluence Environment to use by default with this project (Use arrow keys)
+❯ kras (default)
+  testnet
+  stage
+  local
```
Comment (folex) [inline docs/build/get-started.mdx:91]: Why is local red? Maybe remove `bash`? confuses focus

File: docs/build/get-started.mdx (hunk 1 above)
Comment (justprosh) [reply docs/build/get-started.mdx]: fixed

File: docs/build/get-started.mdx (hunk 2 above)
Comment (justprosh) [reply docs/build/get-started.mdx:91]: fixed

**Review by folex — APPROVED** [review]

**Review by boneyard93501 — APPROVED** [review]

**Pull request merged**